### ISSUE TITLE
#no-cython-lint directive is not honored

### ISSUE THREAD

**Issue by stefanv** [MarcoGorelli/cython-lint#50]
Version: 0.11 (it would be nice to have `cython-lint --version`!)

Cython file:

```
def _read_string(GenericStream st, size_t n):
    cdef object obj = st.read_string(n, &d_ptr, True)  # no-cython-lint
```

Output:

```
/tmp/foo.pyx:2:17: 'obj' defined but unused
```

**Comment by MarcoGorelli** [comment]
thanks for the report! taking a look

**Comment by MarcoGorelli** [comment]
have addressed both in version 0.12.0 - thanks for the report, much appreciated!

**Comment by stefanv** [comment]
Thanks for the quick fix, @MarcoGorelli!
